pages/employee/index.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+
+    <UForm :validate="validate" :state="employee" class="space-y-4" @submit="submitEmployee" @error="onError">
+        <div class="w-96">
+            <div class="mb-2">
+                <UFormGroup label="First Name" name="first_name">
+                    <UInput placeholder="Enter First Name" v-model="employee.first_name" />
+                </UFormGroup>
+            </div>
+            <div class="mb-2">
+                <UFormGroup label="Last Name" name="last_name">
+                    <UInput placeholder="Enter Last Name" v-model="employee.last_name" />
+                </UFormGroup>
+            </div>
+            <div class="mb-2">
+                <UFormGroup label="Mobile" name="mobile">
+                    <UInput placeholder="Enter Mobile" v-model="employee.mobile" :maxlength="10"
+                        @input="allowNumericValue" />
+                </UFormGroup>
+            </div>
+            <div class="mb-2">
+                <UFormGroup label="Pincode" name="pincode">
+                    <UInput placeholder="Enter Pincode" :maxlength="6" v-model="employee.pincode" @change="checkPincode"
+                        @input="allowNumericValue" />
+                    <span v-if="pincodeError" class="text-red-500">{{ pincodeError }}</span>
```

Comment:
Considering, that there's specific handling needed on the check pincode API, is there a way to populate the errors data that is part of our UForm component manually? If yes, this span tag can be removed.

Reply:
Resolved